docs/toc.yml

```diff
@@ -199,6 +199,8 @@
         href: spfx/web-parts/guidance/getting-started-with-top-actions.md
       - name: Isolated web parts
         href: spfx/web-parts/isolated-web-parts.md
+      - name: Isolated web parts are retired
```

Comment:
should this be more of a statement instead of a status update, like "Isolated web part retirement"?